client/app/dedicated/server/working-status/dedicated-server-working-status.html

```diff
@@ -0,0 +1,34 @@
+<div data-ng-controller="WorkingStatusCtrl as $ctrl">
+
+    <oui-back-button data-ui-sref="app.dedicated.server"
+                     data-title="{{:: 'modal_working_status_title' | translate }}">
+    </oui-back-button>
+
+    <div data-ng-repeat="work in $ctrl.worksDetails track by $index">
+        <h5>
+            <a target="_blank"
+               rel="noopener"
+               title="{{ work.title }} ({{ 'core_new_window' | translate }})"
```

Comment:
Please do not use `title` attribute: https://www.w3.org/TR/html/dom.html#the-title-attribute